### PR TITLE
Add description of venetian blind tilt control configuration

### PR DESCRIPTION
**Pull request by dzukero**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44309
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by dzukero** [comment]
This is for Add RFXtrx Rfy venetian blinds tilt control https://github.com/home-assistant/core/pull/44309.

**Comment by dzukero** [comment]
https://github.com/home-assistant/core/pull/44309

**Comment by MartinHjelmare** [comment]
I've linked the core PR in this PR description.

**Comment by MartinHjelmare** [comment]
Side note: Linking github issues with the `#` shortform doesn't work across repositories.

File: source/_integrations/rfxtrx.markdown [hunk 1]
```diff
@@ -145,6 +145,16 @@ Binary sensors have only two states - "on" and "off". Many door or window openin
...
 
+#### Venetian blind mode
+
+Available only for RFY cover devices. Enables tilt control of venetian blind slats.
+
+Venetian blind motors which control slats tilt can be configured in one of two modes - US (short press of up/down buttons opens/closes the blind, long press controls tilt angle), or European (short press of up/down buttons controls tilt angle, long press opens/closes the blind). You can select one of the following settings depending on your blinds:
+
+- **Unknown** - default, tilt control is not enabled. Leave if the cover is not a venetian blind.
+- **US** - tilt control enabled for blinds in US tilt mode.
+- **EU** - tilt control enabled for blinds in European tilt mode.
```
Comment (frenck) [inline source/_integrations/rfxtrx.markdown]:
```suggestion
#### Venetian blind mode

Available only for RFY cover devices. Enables tilt control of venetian blind slats.

Venetian blind motors that control slats tilt can be configured in one of two modes - US (short press of up/down buttons opens/closes the blind, long-press controls tilt angle), or European (short press of up/down buttons controls tilt angle, long-press opens/closes the blind). You can select one of the following settings depending on your blinds:

- **Unknown** - default, tilt control is not enabled. Leave if the cover is not a venetian blind.
- **US** - tilt control enabled for blinds in US tilt mode.
- **EU** - tilt control enabled for blinds in European tilt mode.
```

**Review by frenck — APPROVED** [review]
Thanks, @dzukero! 👍 

Happy New Year 🥂

**Comment by dzukero** [comment]
@frenck,  Happy New Year!

**Pull request merged**